torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -1281,25 +1281,6 @@ void ComputeConstant(Node* n, int opset_version) {
       break;
     }
     case ::c10::onnx::Gather: {
-      if (ConstantValueMap::HasShapeValue(n->input(0)->debugName()) &&
-          ConstantValueMap::HasValue(n->input(1)->debugName())) {
-        // Special case for pattern Shape -> Gather, to propagate shape value.
-        // Gather input 0 is 1d tensor, Gather input 1 is scalar.
-        // Gather output will be scalar.
-        auto shape_value =
-            ConstantValueMap::GetShapeValue(n->input(0)->debugName()).value();
-        auto idx_value =
-            ConstantValueMap::GetValue(n->input(1)->debugName()).value();
-        // Consider the case when Gather index is a scalar.
-        if (idx_value.dim() == 0) {
-          auto idx_value_0 = idx_value.item<int64_t>();
-          if (idx_value_0 >= 0) {
-            std::vector<c10::ShapeSymbol> dims = {shape_value.at(idx_value_0)};
-            c10::SymbolicShape symShape(dims);
-            ConstantValueMap::SetShapeValue(n->output()->debugName(), symShape);
-          }
-        }
-      }
       break;
```

Comment:
what's the point of keeping this case here?

Reply:
Removed. Thanks for catching this.